CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## [5.1.3] - 2022-10-03
+
+### Changed
+
+Headers are reduced to bringdown the size of the logs sent to sumologic.
```

Comment:
This library has no relation to Sumologic. It can be used with any logger, that is passed as dependency through constructor https://github.com/Cimpress-MCP/lambda-essentials-ts/blob/main/src/openApi/openApiWrapper.ts#L27

Can you rephrase the changelog entry to something like:
```
RequestLogger now logs only `Host`, 'User-Agent`, `orion-correlation-id-parent`, `orion-correlation-id-root` headers.
```